steps/aws-device-farm-runner/0.0.2/step.yml

```diff
@@ -0,0 +1,112 @@
+title: Amazon Device Farm Runner
+summary: Amazon Device Farm Runner
+description: |-
+  Deploys app to device farm and starts a test run with a preconfigured test package and device pool.
+
+  This Step requires an Amazon Device Farm registration. To register an account, [click here](https://aws.amazon.com/device-farm/)
+website: https://github.com/peartherapeutics/bitrise-aws-device-farm-runner
+source_code_url: https://github.com/peartherapeutics/bitrise-aws-device-farm-runner
+support_url: https://github.com/peartherapeutics/bitrise-aws-device-farm-runner/issues
+published_at: 2016-09-22T14:38:45.966397965-07:00
+source:
+  git: https://github.com/peartherapeutics/bitrise-aws-device-farm-runner.git
+  commit: bb8ef98aa3ab32563137a1d7411b34b82f622647
```

Comment:
no problem, we have these checks exactly for this reason, to see & let you know when this happens ;)

the hash itself is based into the yml during sharing to "pin" the exact state of the code, so that if we `git clone` the same tag e.g. next month / in a year from now we'll still get the exact same state of the code (which is stored on our CDN as well)